Description: define gettext version to look for
 It needs to be defined.
Author: Laszlo Boszormenyi (GCS) <gcs@debian.org>
Forwarded: no
Last-Update: 2015-11-02

---

--- dar-2.5.1.orig/configure.ac
+++ dar-2.5.1/configure.ac
@@ -8,7 +8,7 @@ AC_DEFINE_UNQUOTED(DAR_VERSION, "AC_PACK
 
 AM_INIT_AUTOMAKE([subdir-objects])
 AM_GNU_GETTEXT([external])
-AM_GNU_GETTEXT_VERSION
+AM_GNU_GETTEXT_VERSION(0.19)
 XGETTEXT_EXTRA_OPTIONS='--keyword=dar_gettext'
 AM_ICONV
 
